wp-content/themes/access/views/locations/single.twig

```diff
@@ -13,20 +13,6 @@
     </div>
     {% endif %}
 
-    <section>
-      <iframe id="google-maps-container"
-        class="block bg-blue-light border border-grey-light print:hidden screen-desktop:hidden"
-        width="100%" height="300" frameborder="0"
-        src="about:blank"></iframe>
-    </section>
```

Comment:
@devowhippit I noticed there were 2 iframe elements but I don't think this one is being used so I deleted.

Reply:
@theasteve one is for mobile and one is for desktop. This was the mobile iframe, the `screen-desktop:hidden` class hides it on desktop.

Reply:
Double-check that the iframe shows up. I don't see a class that hides the other iframe.